packages/credentials/tests/presentation-exchange.spec.ts

```diff
@@ -1,3 +1,4 @@
+import fs from 'fs';
```

Comment:
The fs package isn't available in web browsers so browser tests are failing.

Suggestion:

1. Remove 'fs'
```suggestion
```

2. Add import for the file:
```typescript
import { VerifiableCredential } from '../src/verifiable-credential.js';
import { PresentationExchange } from '../src/presentation-exchange.js';
import PresentationExchangeSelectCredentialsSpec from '../../../test-vectors/presentation_exchange/select_credentials.json' assert { type: 'json' };
```

3. Change test to use the imported test vector:

```typescript
  describe('Web5TestVectorsPresentationExchangeSpec', () => {
    it('select_credentials', async () => {
      const vectors = PresentationExchangeSelectCredentialsSpec.vectors;
      ...
```